diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7948825..acbb352 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,5 +20,3 @@ pico_enable_stdio_uart(display_driver 0)
 # create map/bin/hex file etc.
 pico_add_extra_outputs(display_driver)
 
-# add url via pico_set_program_url
-example_auto_set_url(display_driver)
diff --git a/display.h b/display.h
index b27fb1b..9bd6fc7 100644
--- a/display.h
+++ b/display.h
@@ -4,6 +4,7 @@
 #include <stdlib.h>
 #include <iostream>
 #include <string>
+#include <cstdint>
 
 class Display
 {
diff --git a/sharp-mip/sharp_mip_display.cpp b/sharp-mip/sharp_mip_display.cpp
index 3e91915..77b59e0 100644
--- a/sharp-mip/sharp_mip_display.cpp
+++ b/sharp-mip/sharp_mip_display.cpp
@@ -1,3 +1,4 @@
+#include "pico/stdlib.h"
 #include "sharp_mip_display.h"
 
 SharpMipDisplay::SharpMipDisplay(uint16_t width, uint16_t height, spi_inst_t* spi, uint display_cs_pin)
@@ -301,4 +302,4 @@ void SharpMipDisplay::PrintBinaryArray(const uint8_t* array_to_print, size_t wid
         }
         printf("\n");
     }
-}
\ No newline at end of file
+}
